packages/across-monitor/src/index.ts

```diff
@@ -25,21 +25,29 @@ export async function run(logger: winston.Logger, l1Web3: Web3): Promise<void> {
 
     // l1Client uses bridgeAdmin contract for bridge pool discovery.
     const bridgeAdminAddress = await getAddress("BridgeAdmin", config.bridgeAdminChainId);
-    const l1Client = new InsuredBridgeL1Client(logger, l1Web3, bridgeAdminAddress, null);
+    const l1Client = new InsuredBridgeL1Client(
+      logger,
+      l1Web3,
+      bridgeAdminAddress,
+      null,
+      config.bridgePoolEarliestBlockToSearch,
+      null, // Use latest block as end block
+      config.bridgePoolMaxBlocksToSeach
+    );
 
     const acrossMonitor = new AcrossMonitor(logger, config, l1Client);
 
     for (;;) {
       await retry(
         async () => {
           // Updating acrossMonitor also updates l1Client for pool discovery.
-          await acrossMonitor.update();
+          acrossMonitor.update();
```

Comment:
I'm slightly confused. now that `acrossMonitor.update()` no longer pulls in the event data and rather reads everything from the `l1Client` via `this.l1Client.allRelayEventData`, where is `await l1Client.update()` being called? i.e where is the monitor now actually pulling in the event data, via the l1Client?